ResearchKit/Localized/en.lproj/Localizable.strings

```diff
@@ -216,6 +216,15 @@
 "REACTION_TIME_TASK_ATTEMPTS_FORMAT" = "Attempt %@ of %@";
 "REACTION_TIME_TASK_ACTIVE_STEP_TITLE" = "Quickly shake the device when the blue circle appears";
 
+/* Tower of Hanoi active task */
+"TOWER_OF_HANOI_TASK_TITLE" = "Tower of Hanoi";
+"TOWER_OF_HANOI_TASK_INTENDED_USE" = "This activity evaluates your puzzle solving skills.";
+"TOWER_OF_HANOI_TASK_TASK_INTRO_TEXT_FORMAT" = "Move the entire stack to the highlighted platform in as few moves as possible.";
+"TOWER_OF_HANOI_TASK_TASK_CALL_TO_ACTION" = "Tap Get Started to begin";
+"TOWER_OF_HANOI_TASK_ACTIVE_STEP_INTRO_TEXT" = "Solve the Puzzle \n";
```

Comment:
A little dirty.. when the title label's text changes from "solve the puzzle" to "number of moves \n 00:00". If the number of lines changes, the content size of the title changes and the active step view below it, which contains the towers, moves down when the text changes, which doesn't look very nice.